src/app/external-cluster-wizard/steps/external-cluster/provider/gke/template.html

```diff
@@ -63,20 +66,72 @@
       </mat-error>
     </mat-form-field>
 
-    <mat-form-field>
-      <mat-label>Initial Node Count
-      </mat-label>
-      <input matInput
-             type="number"
-             [formControlName]="Controls.NodeCount" />
-      <mat-hint>Set the Initial Node Count.</mat-hint>
-      <mat-error *ngIf="form.get(Controls.NodeCount).hasError('required')">
-        Initial Node Count is <strong>required</strong>.
-      </mat-error>
-    </mat-form-field>
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Disk Types"
+                     [options]="diskTypes"
+                     [formControlName]="Controls.DiskTypes">
+      <mg-container hint>
+        Set Disk Type
+      </mg-container>
+    </km-autocomplete>
+
+    <km-autocomplete *ngIf="isDialogView()"
+                     label="Machine Sizes"
+                     [options]="machineSizes"
```

Comment:
show `name + description` you need to update options in `ts` file using `map` operator